diffcp/cone_program.py

```diff
@@ -42,29 +55,22 @@ class SolverError(Exception):
     pass
 
 
-def solve_and_derivative(A, b, c, cone_dict, warm_start=None, **kwargs):
+def solve_and_derivative(A, b, c, cone_dict, warm_start=None, mode='lsqr', **kwargs):
```

Comment:
nit: the newlines in this function's docstring were deleted somehow, which IMO makes it less readable. Can you add them back in?

Reply:
Weird, fixed. Maybe a git merge/rebase did this?

Reply:
Unresolving, looks like you forgot to push? (Or maybe I'm looking at the wrong edit, the Github review UI really confuses me)

Reply:
(Sorry, about to push all of these changes at once)